src/pattern-library/components/patterns/UsingComponentsPage.js

```diff
@@ -21,14 +21,23 @@ export default function UsingComponentsPage() {
     >
       <Library.Pattern title="Component categories">
         <ul>
-          <li>
-            <strong>Simple components</strong>: Opinionated, simple components
-            that apply design patterns. Example: <code>Spinner</code>.
-          </li>
           <li>
             <strong>Presentational components</strong>: Composable components
             with customization flexibility. Example: <code>Button</code>,{' '}
-            <code>Link</code>.
+            <code>Link</code>. This is the most common category of component.
+            <ul>
```

Comment:
Indented lists like this could use a bit more vertical spacing around them.

Reply:
Yep! Already tracked, second checklist item here: https://github.com/hypothesis/frontend-shared/issues/587